docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb

```diff
@@ -54,22 +54,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dt = ww.DataTable(df_dask, index='order_product_id')\n",
-    "dt"
+    "df_dask.ww.init(index='order_product_id')\n",
```

Comment:
The text section above this has a reference to "DataTable"

Reply:
Fixed